etc/kayobe/pulp-repo-versions.yml

```diff
@@ -45,6 +45,6 @@ stackhpc_pulp_repo_rocky_9_1_crb_version: 20230228T044432
 stackhpc_pulp_repo_rocky_9_1_extras_version: 20230228T044432
 stackhpc_pulp_repo_rocky_9_1_highavailability_version: 20230228T044432
 stackhpc_pulp_repo_treasuredata_4_version: 20221105T035018
-stackhpc_pulp_repo_ubuntu_cloud_archive_version: 20230424T064949
-stackhpc_pulp_repo_ubuntu_focal_security_version: 20230424T064949
-stackhpc_pulp_repo_ubuntu_focal_version: 20230424T064949
+stackhpc_pulp_repo_ubuntu_cloud_archive_version: 20230515T073310
+stackhpc_pulp_repo_ubuntu_focal_security_version: 20230515T073310
+stackhpc_pulp_repo_ubuntu_focal_version: 20230515T073310
```

Comment:
Bumping all Ubuntu snapshots seems like something we should mention in a reno

Reply:
Done.